code/game/mecha/mecha.dm

```diff
@@ -639,6 +639,7 @@
 			AI.control_disabled = 1
 			AI.radio_enabled = 0
 			AI.loc = card

```

Comment:
you don't need to both update the loc and use forceMove

force move does it and handles other things like camera view, in general it is preferred over simply setting the loc for that reason.